### PR TITLE
add Aeraki to integrations

### PR DESCRIPTION
**Pull request by zhaohuabing**
Signed-off-by: zhaohuabing <zhaohuabing@gmail.com>

add Aeraki to ecosystem - integrations

https://github.com/aeraki-framework

Introduction: https://istio.io/latest/blog/2021/aeraki/

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

- [ ] Configuration Infrastructure
- [ ] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Policies and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

File: data/companies.yml [hunk 1]
```diff
@@ -228,6 +228,12 @@ integrations:
     description: "Space Cloud is a Kubernetes based serverless platform that provides instant, realtime APIs on any database, with event triggers and unified APIs for your custom business logic."
     details:
       - "Space Cloud integrates with Kubernetes and Istio natively to bring to you a highly scalable Serverless Platform. It encrypts all traffic by default and lets you describe communication policies to protect your microservices."
+  - name: "Aeraki"
+    logo: "/logos/aeraki.png"
+    url: "https://github.com/aeraki-framework/"
+    description: "Aeraki extends Istio to manage traffic for any layer-7 protocols."
+    details:
+      - "Aeraki [Air-rah-ki] is the Greek word for ‘breeze’. While Istio connects microservices in a service mesh, Aeraki provides a framework to allow Istio to support more layer-7 protocols other than HTTP and gRPC. We hope this breeze can help Istio sail a little further."
```
Comment (craigbox) [inline data/companies.yml:236]: Please sort this in alphabetical order.

(Do you have an SVG image instead of a PNG by any chance?)

**Review by craigbox — CHANGES_REQUESTED** [review]

File: data/companies.yml (hunk 1 above)
Comment (zhaohuabing) [reply data/companies.yml]: Rearranged it in alphabetical order.

The original one is in PNG format. I tried to convert it to SVG, but the SVG image is too coarse.

**Review by Arhell — COMMENTED** [review]
lgtm

**Pull request merged**